packages/loader/container-utils/src/error.ts

```diff
@@ -3,27 +3,26 @@
  * Licensed under the MIT License.
  */
 
-import { assert } from "@fluidframework/common-utils";
 import {
     ContainerErrorType,
     IGenericError,
     ICriticalContainerError,
     IErrorBase,
+    IThrottlingWarning,
 } from "@fluidframework/container-definitions";
-import { LoggingError } from "@fluidframework/telemetry-utils";
+import { isILoggingError, LoggingError } from "@fluidframework/telemetry-utils";
 import { ITelemetryProperties } from "@fluidframework/common-definitions";
 import { ISequencedDocumentMessage } from "@fluidframework/protocol-definitions";
 
 function messageFromError(error: any): string {
     if (typeof error?.message === "string") {
-        // eslint-disable-next-line @typescript-eslint/no-unsafe-return
-        return error.message;
+        return error.message as string;
     }
-    return `${error}`;
+    return String(error);
 }
 
-const isValidLoggingError = (error: any): error is LoggingError => {
-    return typeof error?.errorType === "string" &&  LoggingError.is(error);
+const isValidLoggingError = (error: any): error is LoggingError & IErrorBase => {
```

Comment:
Since you are poking at it, can you please consider giving it different name to better differentiate from 
isILoggingError?
"valid" is useless (I think it should be removed), and the rest looks almost identical to the other one, making it hard to see the difference.

Reply:
Yeah I've been pondering about when we should be pivoting on an object being a concrete `LoggingError` instance (as best we can) v. adhering to `ILoggingError`. I wrote `isILoggingError` and then @anthony-murphy wrote `LoggingError.is`.

In this case, "valid" I think refers to the fact that it also has `errorType`.  I will take a moment to think over a better name/factoring.

Reply:
Added a comment. it needs to stay as-is